internal/backend/scheduler/scheduler.go

```diff
@@ -109,11 +113,29 @@ func (sch *Scheduler) Update(ctx context.Context, tid sdktypes.TriggerID, schedu
 func (sch *Scheduler) workflow(wctx workflow.Context, tid sdktypes.TriggerID) error {
 	sl := sch.sl.With("trigger_id", tid)
 
-	event := sdktypes.NewEvent(tid).WithType("tick")
-
 	ctx := temporalclient.NewWorkflowContextAsGOContext(wctx)
 
-	eid, err := sch.dispatcher.Dispatch(ctx, event, nil)
+	// It's ok to run all these straight in the workflow - they're all should be
```

Comment:
disagree
under load it can take time
and it will cause dead lock fails

should be in an activity, same as the delete schedule

Reply:
if it's stuck, there'll be panic and the workflow will retry. but i get it - will turn these into activities.

Reply:
fixed